src/Symfony/Component/Console/Tests/Fixtures/application_run1.txt

```diff
@@ -12,5 +12,6 @@ Options:
   -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
 
 Available commands:
-  help  Display help for a command
-  list  List commands
+  completion  Dump the shell completion script
```

Comment:
or shouldn't we hide the command (and document it elsewhere?), so that it doesn't pollute the list, since it's a use-once command?

Reply:
Use once every time we change something. I think it should be kept in the listing. Or create a composer lock like behaviour. Detecting that the dump should be re-do when using `console`

Reply:
If we somehow manage to make this autoupdateable, we could hide the command.